### PR TITLE
refactor: Extract test data for NPM and PEP440 version schemes

### PR DESCRIPTION
**Pull request by zharinov**
Ref #5213

### CODE REVIEW

**Comment by zharinov** [comment]
So that we can merge it for `poetry`.

**Review by viceice — APPROVED** [review]
Just test code

**Comment by rarkins** [comment]
@zharinov is this still required?

File: lib/versioning/pep440/index.ts [hunk 1]
```diff
@@ -31,7 +31,8 @@ const isStable = (input: string): boolean => {
 };
 
 // If this is left as an alias, inputs like "17.04.0" throw errors
-export const isValid = (input: string): string => validRange(input);
+export const isValid = (input: string): string =>
```
Comment (rarkins) [inline lib/versioning/pep440/index.ts]: Let's not change any pep440 behaviour

**Review by rarkins — CHANGES_REQUESTED** [review]

**Pull request closed without merging**